package.json

```diff
@@ -63,10 +63,13 @@
     "prosemirror-model": "^1.18.3",
     "prosemirror-schema-list": "^1.2.2",
     "prosemirror-state": "^1.4.2",
+    "prosemirror-utils": "^0.9.6",
     "prosemirror-transform": "^1.7.0",
+    "prosemirror-tables": "^1.3.5",
     "prosemirror-view": "^1.29.1"
   },
   "resolutions": {
-    "eslint-mdx": "2.0.5"
+    "eslint-mdx": "2.0.5",
+    "prosemirror-state": "^1.4.2"
```

Comment:
why is that needed?

Reply:
`prosemirror-tables` use old version of state